src/helpers/vyos-load-balancer.py

```diff
@@ -0,0 +1,296 @@
+            run(f'ip route replace table {table_num} default dev {ifname} via {dhcp_nexthop_addr}')
+
+    if_addr = get_ipv4_address(ifname)
+    if if_addr and if_addr != lb['health_state'][ifname]['if_addr']:
+        lb['health_state'][ifname]['if_addr'] = if_addr
+        return True
+
+    return False
+
+def nftables_update(lb):
+    # Atomically reload nftables table from template
+    if not os.path.exists(nftables_wlb_conf):
+        lb['first_install'] = True
+    elif 'first_install' in lb:
+        del lb['first_install']
+
+    render(nftables_wlb_conf, 'load-balancing/nftables-wlb.j2', lb)
+
+    rc = run(f'nft -f {nftables_wlb_conf}')
+
+    if rc != 0:
+        print('ERROR: Failed to apply WLB nftables config')
+        return False
+
+    return True
+
+def cleanup(lb):
+    if 'interface_health' in lb:
+        index = 1
+        for ifname, health_conf in lb['interface_health'].items():
+            table_num = lb['mark_offset'] + index
+            run(f'ip route del table {table_num} default')
+            run(f'ip rule del fwmark {hex(table_num)} table {table_num}')
+            index += 1
+
+    run(f'nft delete table ip vyos_wanloadbalance')
+
+def get_config():
+    conf = Config()
+    base = ['load-balancing', 'wan']
+    lb = conf.get_config_dict(base, key_mangling=('-', '_'),
+                            get_first_key=True, with_recursive_defaults=True)
+
+    lb['test_defaults'] = get_defaults(base + ['interface-health', 'A', 'test', 'B'], get_first_key=True)
+
+    return lb
+
+if __name__ == '__main__':
+    while commit_in_progress():
+        print("Notice: Waiting for commit to complete...")
+        time.sleep(1)
+
+    lb = get_config()
+
+    lb['health_state'] = {}
+    lb['mark_offset'] = 0xc8
+
+    # Create state dicts, interface address and nexthop, install routes and ip rules
+    if 'interface_health' in lb:
+        index = 1
+        for ifname, health_conf in lb['interface_health'].items():
+            table_num = lb['mark_offset'] + index
+            lb['health_state'][ifname] = {
+                'if_addr': get_ipv4_address(ifname),
+                'failure_count': 0,
+                'success_count': 0,
+                'last_success': 0,
+                'last_failure': 0,
+                'state': True,
+                'state_changed': False,
+                'table_number': table_num,
+                'mark': hex(table_num)
+            }
+
+            if health_conf['nexthop'] == 'dhcp':
+                lb['health_state'][ifname]['dhcp_nexthop'] = None
+
+                dynamic_nexthop_update(lb, ifname)
+            else:
+                run(f'ip route replace table {table_num} default dev {ifname} via {health_conf["nexthop"]}')
+
+            run(f'ip rule add fwmark {hex(table_num)} table {table_num}')
+
+            index += 1
+
+        nftables_update(lb)
+
+        run('ip route flush cache')
+
+        if 'flush_connections' in lb:
+            run('conntrack -F')
```

Comment:
Use please `conntrack --delete`
We had bugs with flush https://github.com/vyos/vyatta-wanloadbalance/blob/5f6d38762d25267f974f8035ed79a0369df710c4/src/lbdecision.cc#L297-L300

https://vyos.dev/T1311